### ISSUE TITLE
The Microsoft Power BI tutorial directs you to this page for the sample Access database but there's no access database

### ISSUE THREAD

**Issue by JohnnyCullen** [MicrosoftDocs/powerbi-docs#1783]
The Microsoft Power BI tutorial directs you to this page for the sample Access database PowerBI.accb  but there's no access database here, just the Excel file from the first tutorial

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 423bed63-655a-3079-815e-0d9a8c7de44d
* Version Independent ID: ed119c4a-d4f0-7458-fd57-d7247edbd70e
* Content: [Download the Financial Sample workbook for Power BI - Power BI](https://docs.microsoft.com/en-us/power-bi/sample-financial-download#feedback)
* Content Source: [powerbi-docs/sample-financial-download.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/live/powerbi-docs/sample-financial-download.md)
* Service: **powerbi**
* Sub-service: **powerbi-desktop**
* GitHub Login: @maggiesMSFT
* Microsoft Alias: **maggies**

**Comment by maggiesMSFT** [comment]
Hi, @JohnnyCullen -- clearly you are correct. Can you tell us which Microsoft Power BI tutorial directs you to this page for the Access database? I'm not finding it in our files.
Thanks!

**Comment by tmsoliveira** [comment]
For Example:
https://docs.microsoft.com/pt-pt/learn/modules/get-data-power-bi/4-clean-data-query-editor

**Comment by maggiesMSFT** [comment]
Hi, @tmsoliveira -- Thanks! I see the link on that page that goes to this Download the Financial Sample page, but the note says it's to get the Excel file:
"To follow along with the examples in the videos and on this page, download the sample Excel workbook here and import into Power BI Desktop (Get Data > Excel) if you haven't already."
It doesn't mention an Access db. Or am I still missing it?
Thanks!

**Comment by JohnnyCullen** [comment]
It looks like the sample Access Db is large, 10m+ rows, and unfortunately you dont need access to the Access Db to complete the course. 
It woyld be far better if they used a smaller downloadable Access Db so you coyld repeat the tutorial (sorry lab) steps

 
 
  On Mon, 3 Feb 2020 at 22:00, Maggie Sparkman<notifications@github.com> wrote:   
Hi, @tmsoliveira -- Thanks! I see the link on that page that goes to this Download the Financial Sample page, but the note says it's to get the Excel file:
"To follow along with the examples in the videos and on this page, download the sample Excel workbook here and import into Power BI Desktop (Get Data > Excel) if you haven't already."
It doesn't mention an Access db. Or am I still missing it?
Thanks!

**Comment by maggiesMSFT** [comment]
Hi, @JohnnyCullen -- could you post the article that mentions the Access db and has a link to this article?

**Comment by Shvake** [comment]
Access database mentioned in this video, starting from 2.20
https://www.microsoft.com/en-us/videoplayer/embed/RE3wRgY?pid=RE3wRgY-ax-37-id-oneplayer&postJsllMsg=true&autoplay=false&mute=false&loop=false&market=en-us&playFullScreen=false

**Comment by maggiesMSFT** [comment]
@JohnnyCullen and @Shvake -- thanks so much! I've contacted the author of the Learn content, who is going to fix the links in those articles to go to other data sources. I'm closing this issue.
Thanks again!

**Comment by heyrob** [comment]
Thank you for catching this. The module switches between a sample Excel sheet and an Access database. The link to the Access db in the video has been added to the unit content.

**Comment by maggiesMSFT** [comment]
Thanks for fixing the link, @heyrob !

**Comment by martink71** [comment]
Hi MaggiesMSFT
Im unsure how to use these comment bits, but I imagine if something is closed it is said to be fixed.... 
I am doing an online course, 
Create and use analytics reports with Power BI

I too noticed how these tutorial segments are reused within different courses and I too was suckered into, not only downloading the LARGE Access database, but also to even do that I had to download Access Software to be able to import the Access database, I am kind of glad in some ways because the chap in the video describing the import of Access database which is NOT used, is a lot more methodical/slower, and explains to the likes of me who have never used this sort of software before.
The video was helpfull, but alas, the very next section goes to a video with the lad with the Power BI T-Shirt, talking about Excel Financial table, which I downloaded for a second time, thinking it was different from the first.  Im quite a tech savy chap, but I admit to taking over an hour to complete what is a 10 min segment LOL (Im blaming it on a Very slow laptop and downloading large database file .)

The page in the course where You are asked to connect a Large Access Database is
Unit 3 of 9 

https://docs.microsoft.com/en-gb/learn/modules/get-data-power-bi/3-connect-data-sources-power-bi-desktop

Unit 2 of 9 Asks us to download an Excel document ..... This is fine and I got there in the end trying to keep up with the video... (Suggestion that if one gets into difficulty... there is no "close or close document"  like in other software when using the Desktop version of Power BI, but simply creating a new instance of the software and closing the old, would have been very useful to know
https://docs.microsoft.com/en-gb/learn/modules/get-data-power-bi/2-getting-started-power-bi-desktop

Unit 4 of 9 again asks us to download an excel document, which is the SAME as the one downloaded in Unit 2.

LOVING THE COURSE.
